app/assets/stylesheets/beta.css.scss

```diff
@@ -0,0 +1,42 @@
+@import "compass";
+@import "colors";
+
+.beta-signup {
+  margin-top: 30%;
+  width: 100%;
+
+  .fi-laptop {
+    font-size: 1.8em;
+    margin-right: 3px;
+    vertical-align: middle;
+  }
+}
+
+#beta_signin {
+  .button {
+    border: $cmPurple;
+    @include background-image(linear-gradient(bottom top, #8e4487, #653060 ));
```

Comment:
Color literals like `#8e4487` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#653060` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Expected 0 spaces between parentheses instead of 1